### PR TITLE
post-merge update: networks page

### PR DESCRIPTION
**Pull request by jmcook1186**
## Description

updates info on testnets to be relevant post-merge

## Related Issue

 #7075

### CODE REVIEW

File: src/content/developers/docs/networks/index.md [hunk 1]
```diff
@@ -52,7 +50,7 @@ A proof-of-work testnet; this means it's the best like-for-like representation o
 
 #### Goerli {#goerli}
 
-A proof-of-authority testnet that works across clients; an ideal testnet for application developers. Goerli will be the final testnet merged to proof-of-stake before Ethereum Mainnet is merged. This is expected to happen in summer 2022. Goerli is expected to be maintained long term as a proof-of-stake testnet.
+Originally a proof-of-authority testnet, now proof-of-stake. Goerli is expected to be maintained long term as a stable testnet for application developers.
```
Comment (minimalsm) [inline src/content/developers/docs/networks/index.md:53]: We are burying the lede here

```suggestion
Goerli is a proof-of-stake testnet. It is expected to be maintained long-term as a stable testnet for application developers. Before its testnet merge, Goerli was a proof-of-authority testnet.
```

File: src/content/developers/docs/networks/index.md [hunk 2]
```diff
@@ -68,7 +66,7 @@ A proof-of-authority testnet that works across clients; an ideal testnet for app
 
 _Note, [the Ropsten testnet is deprecated](https://github.com/ethereum/pm/issues/460) and will no longer receive protocol upgrades. Please consider migrating your applications to Sepolia or Goerli._
 
-Ropsten was a proof-of-work testnet that went through The Merge to proof-of-stake in May 2022. It can be used to test applications on a merged network, but it is not expected to be maintained long term and is likely to deprecated before summer 2023.
+Ropsten was a proof-of-work testnet that went through The Merge to proof-of-stake in May 2022. It is likely to deprecated soon.
```
Comment (minimalsm) [inline src/content/developers/docs/networks/index.md:69]:
```suggestion
Ropsten is a proof-of-stake testnet. Ropsten will be deprecated in late 2022. Before undergoing The Merge in May 2022, Ropsten was a proof-of-work testnet.
```

File: src/content/developers/docs/networks/index.md [hunk 3]
```diff
@@ -68,7 +66,7 @@ A proof-of-authority testnet that works across clients; an ideal testnet for app
 
 _Note, [the Ropsten testnet is deprecated](https://github.com/ethereum/pm/issues/460) and will no longer receive protocol upgrades. Please consider migrating your applications to Sepolia or Goerli._
```
Comment (minimalsm) [inline src/content/developers/docs/networks/index.md:67]: Ropsten IS deprecated? Or will be deprecated.

File: src/content/developers/docs/networks/index.md [hunk 4]
```diff
@@ -30,15 +30,13 @@ In addition to Mainnet, there are public testnets. These are networks used by pr
...
 
 ETH on testnets has no real value; therefore, there are no markets for testnet ETH. Since you need ETH to actually interact with Ethereum, most people get testnet ETH from faucets. Most faucets are webapps where you can input an address which you request ETH to be sent to.
 
 #### Sepolia {#sepolia}
 
-A proof-of-work testnet; this means it's the best like-for-like representation of Ethereum. Sepolia is expected to undergo The Merge to proof-of-stake in summer 2022. It is not yet certain whether it will then be maintained long term.
+Originally a proof-of-work testnet, now proof-of-stake. Sepolia is still running but it might not be maintained long term.
```
Comment (minimalsm) [inline src/content/developers/docs/networks/index.md:39]:
```suggestion
Sepolia is a proof-of-stake testnet. Although Sepolia is still running, it is not currently planned to be maintained long-term. Before undergoing The Merge in June 2022, Sepolia was a proof-of-work testnet.
```

File: src/content/developers/docs/networks/index.md [hunk 5]
```diff
@@ -52,7 +50,7 @@ A proof-of-work testnet; this means it's the best like-for-like representation o
 
 #### Goerli {#goerli}
```
Comment (minimalsm) [inline src/content/developers/docs/networks/index.md:51]: Let's put Goerli top of this list if it is expected to be the main testnet people use.

**Review by minimalsm — CHANGES_REQUESTED** [review]
Looking good. I think we should reorder the testnets with the most future-proof (Goerli?) being first. Also suggested a small tweak to lead with information about is and not what the testnet was.

File: src/content/developers/docs/networks/index.md (hunk 3 above)
Comment (jmcook1186) [reply src/content/developers/docs/networks/index.md:67]: already deprecated (i.e. no active maintenace), will be shut down in Q4

Comment (minimalsm) [reply src/content/developers/docs/networks/index.md:68]: Ah, the language isn't clear here then. We're using deprecated to mean both: 

- no longer maintained
- no longer accessible

File: src/content/developers/docs/networks/index.md (hunk 4 above)
Comment (q9f) [reply src/content/developers/docs/networks/index.md]: I don't know where this came from but it's factually wrong. Sepolia will be maintained until at least 2026.

https://ethereum-magicians.org/t/proposal-predictable-ethereum-testnet-lifecycle/11575/5

**Pull request merged**